helpers/github_helpers.rb

```diff
@@ -5,8 +5,10 @@ def build_contributors(repo)
     return unless config[:github_auth]
     @@github = Github.new do |c|
       c.basic_auth = config[:github_auth]
+      c.connection_options = {headers: {"X-GitHub-OTP" => config[:github_tfa]}}
     end
-    @@repo_contributors = @@github.repos.contributors 'ind9', repo
+    print(", " + repo)
```

Comment:
Yeah. auth is meant for deploy time, and client side rendering for dev time. If you set auth in dev, it will fetch info server side for every reload.